cmd/vic-machine/inspect/inspect.go

```diff
@@ -112,6 +113,15 @@ func (i *Inspect) Run(cli *cli.Context) error {
 	}
 	executor.InitDiagnosticLogs(vchConfig)
 
+	installerVer := version.GetBuild()
+
+	log.Info("")
```

Comment:
because then it's not aligned correctly:
```
INFO[2016-09-12T09:52:23-07:00] ### Inspecting VCH ####                      
INFO[2016-09-12T09:52:23-07:00]                                              
INFO[2016-09-12T09:52:23-07:00] VCH ID: VirtualMachine:94                    
INFO[2016-09-12T09:52:23-07:00] 
Installer version: v0.5.5--3d3dd05          
INFO[2016-09-12T09:52:23-07:00] VCH version: v0.5.5-124-216f97c              
INFO[2016-09-12T09:52:23-07:00] 
VCH upgrade status:                         
INFO[2016-09-12T09:52:23-07:00] Upgrade in progress 
```